misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -63,6 +64,10 @@ class JettyService @Inject internal constructor(
     }
     httpConnectionFactories += HttpConnectionFactory(httpConfig)
 
+    if (webConfig.http2) {
```

Comment:
Does this break HTTP/1 over plaintext? It would be better to create a new http2c config?

Reply:
... and potentially remove the regular HTTP config there.

Reply:
HTTP/1 still works since this appends the h2c connection factory instead of replacing the HTTP/1 factory. Given that, why have a second Misk config?